client/cgo/cgotest.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+/*
+int testVRFVerify(unsigned char *pk, int pkSize,
+    unsigned char *m, int mSize,
+    unsigned char *index, int indexSize,
+    unsigned char *proof, int proofSize) {
+    if (cgoVrfVerify(pk, pkSize, m, mSize,
```

Comment:
No, it's not what I mean, haha. I mean my use of something like unsafe.Pointer and others ...
Ok, I agree that they could be simplified as you said.